### ISSUE TITLE
Registry mode

### ISSUE THREAD

**Issue by h0munculus** [snowie2000/mactype#865]
Hello, long time user of MacType. Well, since gdipp / GDI++ times at least. In the latest version Registry mode does not work, even though I followed the guide. With every reboot the rendering is off, and I'd have to enable every single process (that works with MacType) over and over again. MacType 2019.1b6 works fine though, but then again it also supported Registry mode natively?

Reason for wanting to use Registry mode is simple, it's the only one that gives me flawless results.

**Comment by snowie2000** [comment]
Are you sure the version 2019.1b6 still works well in your system with register mode?

The fact is that when we have a feature that works well sometimes and makes a lot of trouble otherwise, and we have not figured out a way to address any of them, what's would the best way to deal with it? We decided to keep it, but make it less easy to reach. 
That is the registry mode.

**Comment by CaiAcmer** [comment]
Maybe you should turn off secure boot in bios

**Comment by h0munculus** [comment]
> 

Yes. Better than the latest one here on Github now, as I can't make use of the Registry Mode. I've tried Service Mode and the MacTray modes, the latter one is **convenient** , I'll give you that, but I get strange artifacts where parts doesn't render correctly, just inpartial. So I'm one of those Registry mode die-hards still..

> Maybe you should turn off secure boot in bios

Goes without saying, of course it's turned off. I don't miss stuff like that.

The problem with the latest RC version of MacType and you go through the hoops to activate it is that after each reboot MacType is not rendering fonts, because it forgot all the processes it's supposed to render!

**Comment by h0munculus** [comment]
So, I decided to give it another go, so I uninstalled and cleaned up as much of the remains as possible and did everyhing from scratch, didn't care about losing the custom profiles I setup with the MacType Tuner I made. Then installed latest MacType again, followed the Registry Mode tutorial in the Wiki here, and now it works! Something must have been borked before, but now it works again!
